src/components/Widgets/DateTimeControl.js

```diff
@@ -4,8 +4,12 @@ import DateTime from 'react-datetime';
 
 export default class DateTimeControl extends React.Component {
   componentDidMount() {
-    if (!this.props.value) {
-      this.props.onChange(new Date());
+    const { value, onChange } = this.props;
+    if (value == null) {
```

Comment:
I have a personal issue with the `== null` approach, and it'll be flagged whenever we re-enable the linter. I'd recommend putting the false case first, and just following that with the existing falsey check.